entrypoint.sh

```diff
@@ -25,9 +26,30 @@ if [ -n "$MYSQL_PORT_3306_TCP" ]; then
 		echo >&2 "  Connecting to DB_HOST ($DB_HOST)"
 		echo >&2 '  instead of the linked mysql container.'
 	fi
-fi
+	: ${DB_USER:='root'}
+	if [ "$DB_USER" = 'root' ]; then
+		: ${DB_PASS:=$MYSQL_ENV_MYSQL_ROOT_PASSWORD}
+	fi
+	: ${DB_PORT:='3306'}
+elif [ -n "$POSTGRES_PORT_5432_TCP" ]; then
+	DB_TYPE=pgsql
+	if [ -z "$DB_HOST" ]; then
+		DB_HOST=$POSTGRES_PORT_5432_TCP_ADDR
+		DB_PORT=$POSTGRES_PORT_5432_TCP_PORT
+		: ${DB_NAME:='postgres'}
```

Comment:
please remove that line, it will be set in line 61 to the default `drupal` if necessary.